src/test/java/com/uber/jenkins/phabricator/coverage/CoberturaXMLParserTest.java

```diff
@@ -117,6 +106,20 @@ public void testGetLineCoverageWithIncludes()
 
     }
 
+    @Test
+    public void testGetLineCoverageWithMultipleIncludes()
```

Comment:
@ChaitanyaPramod This test checks the case you mentioned with multiple files having same name but different package. I only added it for cobertura as the test resource was easy to obtain. Feel free to add a test after this merges for the jacoco use case

I will be adding a test for the jacoco file provider i.e the one which does not rely on jekins jacoco plugin in a follow up change